.travis.yml

```diff
@@ -8,6 +8,8 @@ matrix:
     - php: 7.1
     - php: 7.1
       env: SYMFONY_VERSION='2.8.*'
```

Comment:
Please do 2 additional things:

- add testing on PHP 7.2
- add a job forcing testing on the 3.4 LTS, similar to this 2.8 job

Reply:
agreed